### ISSUE TITLE
Windows Update always updates driver

### ISSUE THREAD

**Issue by Remocay** [pal1000/Realtek-UAD-generic#78]
I'm currently having an issue with Windows Update, where it always updates the driver as soon as it gets an internet connection. pal1000's driver works perfectly, I'm able to change the settings with the Realtek Audio Console, until the driver is "updated", and I can't open the Console anymore because it says that it's not compatible. Is there any way to stop Windows Update from doing that?

https://i.imgur.com/EcUS1ku.png
https://i.imgur.com/S2sfULt.png
https://i.imgur.com/uZgi1yX.png
Here are some screenshots of the drivers in Driver Store Explorer.

**Comment by pal1000** [comment]
See #48 and #45.

**Comment by Remocay** [comment]
I installed the registry key in the #45 post, and it seems that it worked!
I will keep this issue open for a few days in case anything happens.
Thank you pal1000!

**Comment by Remocay** [comment]
I've used the driver and the registry key, and it has worked perfectly these last few days.

If anyone has the same problem, what I did was the following: Follow all the [steps in here](https://github.com/pal1000/Realtek-UAD-generic#switching-from-other-realtek-hd-audio-drivers), after step 9, don't reconnect to the internet yet and install this [registry key](https://github.com/pal1000/Realtek-UAD-generic/files/4830493/disable-audio-driver-update.zip) (found in #45). When you have the reg key installed don't get scared if Windows Update manages to download the driver, in my case when it tried to install the driver it failed.

Thanks again pal1000!
